### PR TITLE
[Feat] document work calendar guide

### LINKED ISSUE #127: Explain calendar event hierarchy
- The hierarchy scope levels
- The semaphore pattern

### LINKED ISSUE #129: Guide to create  calendar in GQL
1.  Add equipment.
2. Add hierarchyscope
3. Add workCalendar definition

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
- Explainer closes #127
- Guide closes #129

### CODE REVIEW

**Comment by MattDodsonEnglish** [comment]
@dbecker00 The part about overrides needs to be reworked a bit.

File: content/how-to/work-calendars/create-work-calendar.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+{{< /expandable >}}
+
+
+### Add Hierarchy scope
+
+The hierarchy scope establishes the calendar hierarchy that the Rhize calendar service uses to establish [calendar precedence]({{< relref "about-calendars-and-overrides" >}}).
+
+
+**Requirements.**
+A calendar hierarchy scope must have the following preperties.
```
Comment (MattDodsonEnglish) [inline content/how-to/work-calendars/create-work-calendar.md:171]:
```suggestion
A calendar hierarchy scope must have the following properties.
```

File: content/how-to/work-calendars/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+categories: "how-to"
+description: How to configure work calendars to account for planned and unplanned downtime in your operation.
+weight: 500
+menu:
+  main:
+    parent: how-to
+    identifier: howto-work-calendars
+---
+
+Work calendars represent planned and unplanned downtime in your operation.
```
Comment (germanandy) [inline content/how-to/work-calendars/_index.md:14]: Work calendars only represent planned downtime.

File: content/how-to/work-calendars/about-calendars-and-overrides.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  
+
+### Calendar states
+
+The Rhize database and service has three calendar types:
+
+- `PlannedBusyTime`
+- `PlannedDowntime`
+- `PlannedShutdown`
+
```
Comment (germanandy) [inline content/how-to/work-calendars/about-calendars-and-overrides.md:75]: The Rhize database and service has three calendar entry types:


- `PlannedDowntime`
- `PlannedShutdown`
-  'Other'

Other is used to place calendar events into the database which will not be considered when calculating OEE

File: content/how-to/work-calendars/_index.md (hunk 2 above)
Comment (MattDodsonEnglish) [inline content/how-to/work-calendars/_index.md:14]:
```suggestion
Work calendars represent planned downtime in your operation.
```

**Pull request merged**